src/components/Grid/Column.js

```diff
@@ -63,4 +63,6 @@ Column.defaultProps = {
   xLarge: null
 };
 
+Column.displayName = "Column";
```

Comment:
I don't think we need to hard code names here. We can use https://github.com/styled-components/babel-plugin-styled-components to see component names in development. Same comment applies to other files modified.

Reply:
@ooHmartY The reason I added displayName is for the brand site. So if the component not functional or class component when we display the component name in the code section it says styled.div. When add the diaplayName it appears properly in the code snippet.

Reply:
👍